docs/permissions/permission_use_cases.md

```diff
@@ -239,4 +239,29 @@ users without permissions for content can still manage products.
 
 - `product/create`
 - `product/view`
-- `product/edit`
\ No newline at end of file
+- `product/edit`
+
+## Commerce [[% include 'snippets/commerce_badge.md' %]]
+
+You can control the extent of functionalities that are available to store users, 
+by component.
+
+### Cart 
+
+Set the following permissions to decide what actions are available when users 
+interact with carts: 
+
+- `cart/view` - to allow user to view their cart 
+- `cart/delete` - to delete cart, for example, after successful checkout
+- `cart/create` - to create a new cart, for example, after successful checkout
+- `cart/edit` - to allow user to add products to their cart
+
+### Checkout 
+
+Set the following permissions to decide what actions are available when users 
+interact with checkout: 
+
+- `cart/view` - to allow user to view checkout, for example, to block guest buyers access to checkout
+- `cart/create` - to create a new checkout, for example, after workflow completes
+- `cart/update` - to allow user to change product quantity
+- `cart/delete` - to delete checkout, for example, after workflow completes
```

Comment:
Checkout requires `checkout/*` permissions too. This should be discussed with @konradoboza

Reply:
Checkout is not deleted, I think this is a typo - it should be `cart/delete` permission referring to cart deleting which is indeed done on the last checkout step.

Reply:
@konradoboza When is the checkout/delete permission used, when the user backs away from the process and abandons it?

Reply:
Actually, it isn't used at the moment. It will be needed once we start implementing order management - once the order is formed, the checkout needs to be deleted.